qa/tasks/netem.py

```diff
@@ -0,0 +1,272 @@
+"""
```

Comment:
@shyukri could you drop this file from this PR. as netem.py was added in nautilus.